### ISSUE TITLE
aw text object doesn't work with numbers

### ISSUE THREAD

**Issue by gnl** [t9md/atom-vim-mode-plus#66]
Numbered operations on multiple 'aw' words are only applied to the first one:

O[n]e Two Three Four
-> 2daw

Expected VIM behaviour:
[T]hree Four

Actual result:
[T]wo Three Four

P.S. Awesome plugin! Together with some of your other stuff it's even better than VIM in some ways.

**Comment by t9md** [comment]
Yes its not consistent to pure Vim's behavior and should be fixed.
I think similar issue is there for other text-object in vim-mode-plus and also should be fixed.
But current my priority for this is low.
